Source/Core/ShaderConductor.cpp

```diff
@@ -128,7 +128,23 @@ namespace
             const char* functionName = "DxcCreateInstance";
 
 #ifdef _WIN32
-            m_dxcompilerDll = ::LoadLibraryA(dllName);
+            HMODULE hm = NULL;
+            if (GetModuleHandleEx(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS | GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT, (LPCSTR) "DllMain",
```

Comment:
(LPCSTR)"DllMain"
can be changed to
TEXT("DllMain")
to make it compatible to both MBCS and UNICODE. Or just use GetModuleHandleExA without C-style case to DllMain. Same as GetModuleFileName below.

Reply:
Yes, I think that it is interesting.